conans/client/cmake.py

```diff
@@ -40,7 +40,8 @@ def _generator(self):
                         '10': '10 2010',
                         '11': '11 2012',
                         '12': '12 2013',
-                        '14': '14 2015'}
+                        '14': '14 2015',
+                        '15': '15 2017'}
```

Comment:
Toolset 14.1 fully compatible with 14, so you can use library compiled in VS2015 with VS2017 and backward.